src/main/java/step3/CarRaceApplication.java

```diff
@@ -0,0 +1,11 @@
+package step3;
+
+import step3.view.InputView;
+
+public class CarRaceApplication {
+    public static void main(String[] args) {
+        RandomMoveStrategy randomMoveStrategy = new RandomMoveStrategy();
```

Comment:
인터페이스로 하는 이유가 무엇일까요?
쉽게 갈아 끼우기 쉽게 하기 위함이 아닐까요?
현재는 다른 MoveStrategy로 생성헤서 넣으면, 에러가 발생할거 같아요.

```suggestion
        MoveStrategy randomMoveStrategy = new RandomMoveStrategy();
```
이런식으로 하는게 더 어울리는게 아닐까요?